pkg/sshd/etc/profile

```diff
@@ -0,0 +1,13 @@
+export CHARSET=UTF-8
+export PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+export PAGER=less
+export PS1='(sshd ns):\w\$ '
+export PS1='\h:\w\$ '
```

Comment:
What would we call the image? `gettysshd`? `sshetty`? Short form might be inappropriate to use in polite company! :-)